tiledb/sm/rest/rest_client_remote.h

```diff
@@ -224,6 +224,15 @@ class RestClientRemote : public RestClient {
   Status post_array_schema_to_rest(
       const URI& uri, const ArraySchema& array_schema) override;
 
+  /**
+   * Post an array creation request to the rest server.
+   *
+   * @param uri Storage URI to use for the created array
```

Comment:
I don't think it's the storage URI here. It's the full `tiledb://` URI that might or might not have storage in it